testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/clients/ClientSettingsTest.java

```diff
@@ -99,6 +100,9 @@ public void crudOIDCPublic() {
     @Test
     @EnableFeature(value = Profile.Feature.ACCOUNT2, skipRestart = true)
     public void alwaysDisplayInAccountConsole() {
+        driver.navigate().refresh();
+        waitForPageToLoad();
```

Comment:
Just one small nit pick. :) You can use [`refreshPageAndWaitForLoad`](https://github.com/keycloak/keycloak/blob/master/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/UIUtils.java#L64-L66).
```suggestion
        refreshPageAndWaitForLoad();
```

Reply:
Done